src/components/homePageMainContent/homePageMainContent.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { render, screen, within } from "@testing-library/react";
+import { props } from "../../../__tests__/data/swimlaneProps";
+import appConfig from "appConfig";
+import HomePageMainContent from "./homePageMainContent";
+
+describe("homePageMainContent", () => {
+  it("renders the SkeletonLoader", () => {
+    (global as any).fetch = jest.fn(() =>
+      Promise.resolve({
+        status: 200,
+        json: () => Promise.resolve({ props }),
+      })
+    ) as jest.Mock;
+
+    const { container } = render(<HomePageMainContent />);
+
+    expect(container.classList.contains("chakra-skeleton"));
```

Comment:
I would not expect this to pass because once the data is fetched (as in line 11), it would not render the skeleton loader. This is also missing the second part of the test so it's giving a false positive. Add `.toBeTruthy()` at the end to verify that what's inside the expect is true.

This test makes sense but remove the passed props on line 11. This should test that the skeleton loader renders when there is no data returned from the API.

Then another another test where you are passing the props and you check for expected swimlane or featured content data on the page. This follows the same pattern you did for `hero.test.tsx`.